x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/utils.test.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { BoostType } from './types';
+import { normalizeBoostValues } from './utils';
```

Comment:
it would be awesome if you wanted to write unit tests for each individual util, but no worries if not (for the sake of speed)

Reply:
Hey, sorry, you're too fast. Was actually in the process of doing so: 79a499ab3c3835ae3e8825d9636d56f41de7f121

Reply:
Whaaaat! No, YOU'RE too fast! 😁